src/components/ui/shadcn-io/dropzone/index.tsx

```
@@ -0,0 +1,202 @@
+};
+
+const maxLabelItems = 3;
+
+export const DropzoneContent = ({
+  children,
+  className,
+}: DropzoneContentProps) => {
+  const { src } = useDropzoneContext();
+
+  if (!src) {
+    return null;
```

ESLint

[error] Delete `;`

(prettier/prettier)